sample-compose/app/build.gradle.kts

```diff
@@ -36,7 +36,7 @@ android {
         minSdk = Versions.ANDROID_MIN_SDK_VERSION
         targetSdk = Versions.ANDROID_TARGET_SDK_VERSION
         versionCode = Versions.ANDROID_VERSION_CODE
-        versionName = Versions.ANDROID_VERSION_NAME
```

Comment:
@doandinh I think we're updating the wrong field sadly 😢

Reply:
I'll create a bug ticket for it 🙏

Reply:
I've created the ticket and will add to the sprint of this month: https://github.com/nimblehq/android-templates/issues/394 🤖

Reply:
Oh, it is another thing. Please assign it to me when you created @Tuubz